manager/integration/deploy/test.yaml

```diff
@@ -29,7 +29,7 @@ rules:
   resources: ["customresourcedefinitions"]
   verbs: ["*"]
 - apiGroups: ["policy"]
-  resources: ["podsecuritypolicies"]
+  resources: ["podsecuritypolicies", "poddisruptionbudgets"]
```

Comment:
Why we need `poddisruptionbudgets`? will our test operate PDB directly?

Reply:
This follows the CA test case PR: https://github.com/longhorn/longhorn-tests/pull/936/files#diff-deb01c8f2cd2d469bfeed0826ebac624022e4b668a4f7d1d2f04d4d9c22d5f82R32, not sure if it's necessary.

Reply:
This doesn't show anymore, because I rebased this PR.

Reply:
I've tried with or without `poddisruptionbudgets`, but can't tell the differences. @c3y1huang can you help to elaborate more? Thanks!